R/resource_update.R

```diff
@@ -108,6 +114,7 @@ resource_update <- function(id, path, url = get_default_url(),
     last_modified =
       format(Sys.time(), tz = "UTC", format = "%Y-%m-%d %H:%M:%OS6"),
     url = "update")
+  body <- c(body, extras)
```

Comment:
it'd be good to check that `extras` is in fact a list, could you add `assert(extras, "list")` at the top of the function (to fail early)

Reply:
done